health/health_log.c

```diff
@@ -358,54 +361,100 @@ inline void health_alarm_log(
         const char *units,
         const char *info,
         int delay,
-        uint32_t flags
+        uint32_t flags,
+        uint32_t warn_repeat_every,
+        uint32_t crit_repeat_every
 ) {
     debug(D_HEALTH, "Health adding alarm log entry with id: %u", host->health_log.next_log_id);
```

Comment:
This debug needs to be moved now, since it depends on whether create_new_alarm_entry will be true or not.

Reply:
OK, I'll do that.

Reply:
These codes are not longer part of the change.